content/en/docs/concepts/cluster-administration/node-autoscaling.md

```diff
@@ -0,0 +1,264 @@
+other conditions (for example the time elapsed since a Node was created), in order
+to optimize different properties (for example the maximum lifespan of Nodes in a
+cluster).
+
+The details of how consolidation is performed depend on the configuration of a
+given autoscaler. Consult [Configure workloads for autoscaling](#workload-config)
+for common workload-level configuration options.
+
+{{< note >}}
+Consolidation is formerly known as _scale-down_ in Cluster Autoscaler.
+{{< /note >}}
+
+## Autoscalers {#autoscalers}
+
+The functionalities described in previous sections are provided by Node _autoscalers_.
+In addition to the Kubernetes API, autoscalers also need to interact with cloud provider
+APIs to provision and consolidate Nodes. This means that they need to be explicitly
+integrated with each supported cloud provider. The performance and feature set of
+a given autoscaler can differ between cloud provider integrations.
+
+{{< mermaid >}}
+graph TD
+    na[Node autoscaler]
+    k8s[Kubernetes]
+    cp[Cloud Provider]
+
+    k8s --> |get Pods/Nodes|na
+    na --> |drain Nodes|k8s
+    na --> |create/remove VMs|cp
+    cp --> |get VMs|na
+
+    classDef white_on_blue fill:#326ce5,stroke:#fff,stroke-width:4px,color:#fff;
+    classDef blue_on_white fill:#fff,stroke:#bbb,stroke-width:2px,color:#326ce5;
+    class na blue_on_white;
+    class k8s,cp white_on_blue;
+{{</ mermaid >}}
+
+### Autoscaler implementations
+
+Cluster autoscaler and Karpenter are the two Node autoscalers currently supported by
+[sig-autoscaling](https://github.com/kubernetes/community/tree/master/sig-autoscaling).
+
+From the perspective of a cluster user, both autoscalers should provide a similar Node autoscaling
+experience. Both will provision new Nodes for unschedulable Pods, and both will
+consolidate the Nodes that are no longer optimally utilized.
+
+The most of the difference between the autoscalers is in how they're set up and
+configured, as well as in how well they're integrated with a given cloud provider.
+The autoscalers also provide features outside the Node autoscaling scope described
+on this page, and the features differ between them.
+
+Consult the sections below, and the linked documentation for the individual
+autoscalers to decide which autoscaler fits your use-case better.
+
+#### Cluster Autoscaler
+
+Cluster Autoscaler adds or removes Nodes to pre-configured Node groups.
+Node groups generally map to some sort of provider-side VM group. A
+single instance of Cluster Autoscaler can simultaneously manage multiple
+Node groups. When provisioning, Cluster Autoscaler will add Nodes to the group
+that best fits the requests of pending Pods. When consolidating, Cluster Autoscaler
+always selects specific Nodes to remove as opposed to just resizing the underlying VM group.
+
+Main differences vs Karpenter:
+* Cluster Autoscaler (CA) provides features related to just Node autoscaling. Node lifecycle
```

Comment:
Might be worth calling out the massive cloud provider support CAS has! Karpenter has a lot of work to do to catch up in that regard!